Sources/TripKit/model/TKBookingTypes.swift

```diff
@@ -65,8 +66,29 @@ public enum TKBooking {
       case internalURL
       case externalAction = "externalURL"
       case type
-      case input
       case confirmationMessage
+      case confirmation
+      case input
+    }
+  }
+    
+  public struct ActionConfirmation: Codable, Hashable {
+    public let message: String
+    public let abortActionTitle: String
+    public let confirmActionTitle: String
+    
+    private enum CodingKeys: String, CodingKey {
+      case message
+      case abortActionTitle
+      case confirmActionTitle
+    }
+    
+    public init(from decoder: Decoder) throws {
+      let container = try decoder.container(keyedBy: CodingKeys.self)
+      
+      message = try container.decode(String.self, forKey: .message)
+      abortActionTitle = try container.decode(String.self, forKey: .abortActionTitle)
+      confirmActionTitle = try container.decode(String.self, forKey: .confirmActionTitle)
```

Comment:
We don't need any of that as that's just what the default `Codable` implementation would do, right?

```suggestion
```